scripts/uninstall-zowe.sh

```diff
@@ -145,8 +145,8 @@ if [ ! -f "${CIZT_INSTALL_DIR}/opercmd" ]; then
 fi
 
 ################################################################################
-# stop ZWESIS01
-echo "[${SCRIPT_NAME}] stopping ZWESIS01 ..."
+# stop ZWEXMSTC
+echo "[${SCRIPT_NAME}] stopping ZWEXMSTC ..."
```

Comment:
We may need a similar variable like `KNOWN_ZOWE_JOB_NAMES` to remove all known job names of xmem server. For example, `KNOWN_XMEM_JOB_NAMES="ZWESIS01 ZWEXMSTC"`. Otherwise if the previous installation was starting `ZWESIS01`, this uninstall won't shutdown and remove previous installation properly.

Reply:
Example failure in the log:

```
DELETE 'ZOWEAD3.LOADLIB(ZWESIS01)' >>>
12:28:25 IKJ56****5I DATA SET ZOWEAD3.LOADLIB ALREADY IN USE, TRY LATER
12:28:25 IKJ56****5I DATA SET IS ALLOCATED TO ANOTHER JOB OR USER
12:28:25 IEFA110I DATA SET CONTENTION
12:28:25 DATA SET ZOWEAD3.LOADLIB IN USE BY
12:28:25 SYSNAME  JOBNAME  ASID
12:28:25 S0W1     ZWESIS01 007E
12:28:25 IDC0548I ** MEMBER ZWESIS01 NOT DELETED
12:28:25 IDC0014I LASTCC=8
```